gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py

```diff
@@ -55,7 +55,8 @@
 cfg.CONF.register_opts(service_chain_opts, "simplechain")
 
 # Service chain API supported Values
-sc_supported_type = [pconst.LOADBALANCER, pconst.FIREWALL]
+sc_supported_type = [pconst.LOADBALANCER, pconst.LOADBALANCERV2,
+                     pconst.FIREWALL]
```

Comment:
Put this on hold. We didn't decide whether to add a new service_type for lbaasv2 yet.